config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Encass.java

```diff
@@ -35,6 +35,7 @@
     private Set<EncassArgument> arguments;
     private Set<EncassResult> results;
     private Map<String, Object> properties;
+    private Set<EntityAnnotation> annotations;
```

Comment:
should we add JSONIgnore for this variable ?

Reply:
We are not considering this variable while building the entity.

Reply:
when generating yml from this bean it generates entry for each property. If we dont want a property to be part of yml then we should add JSONIgnore for the variable